queries.js

```diff
@@ -0,0 +1,99 @@
+var promise = require('bluebird');
```

Comment:
This file is not required right since you are implementing functions used by the drivers.